testnet/run_bootstrap.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+set -e
+
+echo "I am a bootstrap container $HOSTNAME node $NET_NODE_KEY"
```

Comment:
I don't see any bootstrapping here, we are just staring a node.

Reply:
I left this file for consistency in the docker compose file. A placeholder for similar environment config as in libp2p case. Will remove if no other bootstrapping will be needed.

Reply:
Yeah, seems pointless and just obscured what's happening.